### PR TITLE
Fix t32bitsrv build conflicts

### PR DESCRIPTION
**Pull request by skitaoka**
- repair t32bitsrv
- fix symbol conflict

### CODE REVIEW

**Comment by skitaoka** [comment]
Jenkins, plz

**Comment by skitaoka** [comment]
Jenkins, test this.

File: toonz/sources/CMakeLists.txt [hunk 1]
```diff
@@ -14,7 +14,11 @@ message("Thirdpary Library Search path:" ${THIRDPARTY_LIBS_HINTS})
 
 if (WIN32)
     message("Windows System")
-    set(QT_PATH "C:/Qt/Qt5.5.1/5.5/msvc2013_64" CACHE PATH "Qt instlattion directory")
+    if(CMAKE_SIZEOF_VOID_P EQUAL 4)
+        set(QT_PATH "C:/Qt/Qt5.5.1_32/5.5/msvc2013")
+    else()
+        set(QT_PATH "C:/Qt/Qt5.5.1/5.5/msvc2013_64")
+    endif()
```
Comment (kusano) [inline toonz/sources/CMakeLists.txt]: `CACHE` is needed for users who installed Qt to the other directory to change `QT_PATH`.

**Comment by skitaoka** [comment]
Jenkins, test this, again.

**Comment by skitaoka** [comment]
Jenkins

**Pull request merged**